src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -28,8 +38,37 @@ const INITIAL_DELETE_DRAFT_STATE = {
   cqlLibrary: null,
 };
 
-export default function CqlLibraryList({ cqlLibraryList, onListUpdate }) {
+function IndeterminateCheckbox({
+  indeterminate,
+  className = "",
+  onChange,
+  id,
+  ...rest
+}: {
+  indeterminate?: boolean;
+} & HTMLProps<HTMLInputElement>) {
+  const ref = React.useRef<HTMLInputElement>(null!);
+
+  React.useEffect(() => {
```

Comment:
not a big deal but this can be added to imports so that no need to access React.useEffect & React.useRef